code/modules/overmap/spacetravel.dm

```diff
@@ -29,18 +38,27 @@ var/list/cached_space = list()
 			return 0
 	return 1
 
+/// uncache the sector and move it to the specified overmap area
+/obj/effect/overmap/visitable/sector/temporary/proc/uncache()
+	cached_space -= "[x]-[y]"
+	z = current_map.overmap_z
+	START_PROCESSING(SSslow_process, src)
+
+/obj/effect/overmap/visitable/sector/temporary/proc/shift_to_cache()
+	z = pick(map_z) // move ourselves into our z-level, where we're kept alive but cannot be reached by overmap objects
+	cached_space["[x]-[y]"] = src
+	STOP_PROCESSING(SSslow_process, src)
```

Comment:
We don't need to actually hide the sector in this way, do we? Just make the overmap effect invisible, set `known` to FALSE and make sure that shuttles' `get_possible_destinations()` can't find destinations where known is set to zero. Or add another variable that does that.

Reply:
we could, but this is way easier and less of a headache, especially now with the new sensor system

Reply:
Disagree, the way you're doing it is a lot more of a hack than the alternative.